pkg/util/log/eventpb/telemetry.proto

```diff
@@ -71,3 +71,31 @@ message CapturedIndexUsageStats {
   bool is_unique = 10 [(gogoproto.jsontag) = ",omitempty"];
   bool is_inverted = 11 [(gogoproto.jsontag) = ",omitempty"];
 }
+
+// CreateChangefeedQuery
```

Comment:
Better comment needed if you going to have a comment.
Also, I'm not too keen on having "query" in its name.  Maybe simply `CreateChangefeed`?